jupyterlab_gcedetails/src/components/hardware_scaling_form.tsx

```diff
@@ -200,11 +202,37 @@ export class HardwareScalingForm extends React.Component<Props, State> {
     this.props.onSubmit(configuration);
   }
 
+  private displayPricingEstimation() {
+    const { details, priceService } = this.props;
+    const { configuration } = this.state;
+
+    const zone = details.instance.zone.split('/').pop();
+    const currentPrice = priceService.getPrice(zone, this.oldConfiguration);
+    const newPrice = priceService.getPrice(zone, configuration);
+    const priceDifference = (newPrice - currentPrice);
+
+    return (
+      <div>
+        <span className={STYLES.subheading}>Pricing Estimation:</span>
+        <div className={STYLES.paragraph}>
+          {`Your updated instance will cost an estimated
+          $${newPrice.toFixed(2)} monthly, an estimated difference of 
```

Comment:
At least to me, difference makes me think that we're always subtracting from the current price. Can we make it say an "increase/decrease of" and then show the absolute value of the difference?